modules/ROOT/pages/index.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-With Anypoint Platform Private Cloud Edition (Anypoint Platform PCE), you run and manage Mule applications on your local servers using your required security policies. With Anypoint Platform PCE, there is no external interaction with your data. Anypoint Platform PCE control plane functionality is locally available and does not interact with runtimes over an internet connection. You configure and run the Anypoint Platform software and you maintain all data storage, processing, and transmission. Refer to the following table for components included in Anypoint Platform PCE. 
+With Anypoint Platform Private Cloud Edition (Anypoint Platform PCE), you run and manage Mule applications on your local servers using your required security policies. With Anypoint Platform PCE, there is no external interaction with your data. Anypoint Platform PCE control plane functionality is locally available. You configure and run the Anypoint Platform software and you maintain all data storage, processing, and transmission. Refer to the following table for components included in Anypoint Platform PCE. 
```

Comment:
I think your version will suffice since it removes the confusion. I took a stab at more clarity, not required so feel free to include or reject:

> With Anypoint Platform Private Cloud Edition (Anypoint Platform PCE), you run and manage Mule applications on your local servers using your required security policies. With Anypoint Platform PCE, the control plane functionality is locally available and does not require interaction with external systems or public internet connectivity. You configure and run the Anypoint Platform software and you maintain all data storage, processing, and transmission. Refer to the following table for components included in Anypoint Platform PCE.